core-services/ci-operator-configresolver/ci-operator-configresolver.yaml

```diff
@@ -53,11 +57,16 @@ items:
     name: ci-operator-configresolver
     namespace: ci
   spec:
+    host: ci-operator-configresolver-ci.svc.ci.openshift.org
```

Comment:
It seems `--validate` is too strict, without give a chance to default value.

Reply:
Does every route need this?

Reply:
Seems so.
Also tried with `route/hook`. Same results.